Stthomas/conceptual (#40593)
dotnet/docs: docs/ai/conceptual/rag.md

---

> title: "Integrate Your Data into AI Apps with Retrival-Augmented Generation"

Reviewer: ```suggestion
title: "Integrate Your Data into AI Apps with Retrieval-Augmented Generation"
```

> description: "Learn how retreival-augmented generation lets you use your data with LLMs to generate better completions in .NET."

Reviewer: ```suggestion
description: "Learn how retrieval-augmented generation lets you use your data with LLMs to generate better completions in .NET."
```

author: catbutler
ms.topic: concept-article #Don't change.
ms.date: 04/15/2024

#customer intent: As a .NET developer, I want to understand how retrieval-augmented generation works in .NET so that LLMs can use my data sources to provide more valuable completions.

---

# Retrieval-augmented generation (RAG) provides LLM knowledge

This article describes how retrieval-augmented generation lets LLMs treat your data sources as knowledge without having to train.

> LLMs have extensive knowledge bases. For most use cases, you can select an LLM with the knowledge you need. But, LLMs don't know about your data unless you've customized them through training. Retrieval-augmented generation lets you make your data available to LLMs without training them on it first.

Reviewer 2: ```suggestion
LLMs have extensive knowledge bases through training. For most scenarios, you can select an LLM that is designed for your requirements, but those LLMs still require additional training to understand your specific data. Retrieval-augmented generation lets you make your data available to LLMs without training them on it first.
```
This part reads a little bit strange to me, can we tweaks the verbiage a bit?  One suggestion above, doesn't have to be that.
